sdk/core/build.gradle

```diff
@@ -9,8 +9,9 @@ ext {
     dropwizardVer = "0.9.2"
     findbugsAnnotationsVer = "3.0.1u2"
     guiceVer = "3.0"
-    jacksonVer = "2.6.3"
     hamcrestVer = "1.3"
+    jacksonVer = "2.6.3"
+    jacksonProtobufVer = "0.9.3"
```

Comment:
Invoked by SerializationUtils. Technically not used yet, will be used soon for serialization of ServiceSpecs.